src/components/organisms/Form/Form.jsx

```diff
@@ -0,0 +1,92 @@
+import { useCallback, useState } from 'react';
+import { Formik, Form as FormikForm, Field } from 'formik';
+import PropTypes from 'prop-types';
+import { Icon } from '../..';
+
+import './Form.scss';
+
+export const Form = ({ placeholder, type, onSubmit }) => {
+    const [error, setError] = useState(false);
+    const submitOnEnter = (event) => {
+        if (event.key === 'Enter' && !event.shiftKey) {
+            event.preventDefault();
+            event.target.form.dispatchEvent(
+                new Event('submit', { cancelable: true, bubbles: true })
+            );
+            setTimeout(() => setError(false), 1000);
```

Comment:
potential memory leak, every timeout should be cleared on componentUnmount

Reply:
Done